pkg/services/ngalert/api/api_provisioning.go

```diff
@@ -259,7 +263,11 @@ func (srv *ProvisioningSrv) RoutePostAlertRule(c *models.ReqContext, ar definiti
 	if err != nil {
 		return ErrResp(http.StatusBadRequest, err, "")
 	}
-	createdAlertRule, err := srv.alertRules.CreateAlertRule(c.Req.Context(), upstreamModel, alerting_models.ProvenanceAPI, c.UserID)
+	provenance, err := determineProvenance(c)
+	if err != nil {
+		return ErrResp(http.StatusBadRequest, err, "")
```

Comment:
Please just check that `err` is returned to the client? I can't remember if its the `err` or `message` argument that is returned in the JSON.